src/client/views/rooms/list.html

```diff
@@ -29,7 +29,7 @@ <h2 class="header" ng-class="{ 'empty': rooms.isEmpty() }">
                 />
             </span>
             <form ng-submit="createRoom()" class="top-right-form">
-                <button type="submit" id="submit">
+                <button type="submit" id="submit" analytics-on="click" analytics-event="Create Room">
```

Comment:
We need both, interface interactions and domain events. For example perhaps nobody use sur submit button and use enter key finally, if it's the case we can remove this button. But we need analytics to know